plugin.css

```diff
@@ -1,65 +0,0 @@
-.pluginContainer {
-        font-family: Arial;
-        padding: 10px;
-    }
-    
-    h1 {
-        font-size: 20px;
-        font-weight: bold;
-        color: grey;
-        padding-bottom: 5px;
-    }
-
-    p {
-        padding-top: 5px;
-        padding-bottom: 5px;
-    }
-    
-    input {
-        border: 1px solid #4676a9;
-        padding-left: 1px;
-        padding-right: 1px;
-        padding-top: 4px;
-        padding-bottom: 4px;
-        margin-top: 7px;
-        margin-bottom: 7px;
-        width: 75px;
-    }
-    
-    input:hover {
-        border: 2px solid #c3e8f4;
-        padding-left: 0px;
-        padding-right: 0px;
-        padding-top: 3px;
-        padding-bottom: 3px;
-        margin-top: 7px;
-        margin-bottom: 7px;
-        outline: none;
-    }
-    
-    input[type=button], input[type=submit] {
-        color: white;
-        background-color: #4676a9;
-        border: 1px solid #4676a9;
-        margin-top: 20px;
-        padding: 7px;
-        outline: none;
-        width: auto;
-    }
-    
-    input[type=button]:hover, input[type=submit]:hover {
-        color: white;
-        background-color: #4676a9;
```

Comment:
it's more efficient to group same styles for different classes, like:

input[type=button]:hover, input[type=submit]:hover,
input[type=button]:focus, input[type=submit]:focus {
  color: white;
  background-color: #4676a9;
  ...
}